sycl/test/basic_tests/image_constructors.cpp

```diff
@@ -0,0 +1,344 @@
+// RUN: %clang -std=c++11 %s -o %t1.out -lstdc++ -lOpenCL -lsycl
+// RUN: env SYCL_DEVICE_TYPE=HOST %t1.out
+// RUN: %clang -std=c++11 -fsycl %s -o %t2.out -lstdc++ -lOpenCL -lsycl
+// RUN: env SYCL_DEVICE_TYPE=HOST %t2.out
+// RUN: %CPU_RUN_PLACEHOLDER %t2.out
+// RUN: %GPU_RUN_PLACEHOLDER %t2.out
+// RUN: %ACC_RUN_PLACEHOLDER %t2.out
+//==------------------- image_constructors.cpp - SYCL buffer basic test
+//----------------==//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
Minor comment: buffer->image. Also, the line 8 and 9 are supposed to be just 1 line, right? i.e.
//==--------- image_constructors.cpp - SYCL image basic test ---------------==//

Reply:
added the change.